components/ArticlesCard.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+import moment from "moment";
+
+export const ArticleCard = ({ article }) => {
+    return (
+        <>
+            <div className="article-item">
+                <h1 className="h1-article">{article.title}</h1>
+                <div className="author-date">
+                    <li>Author: {article.author}</li>
```

Comment:
li tags should always be inside a ul or an ol ad this could be confusing to screen readers